src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -3128,6 +3128,113 @@ def test_ps_s3_persistent_topic_stats():
     time.sleep(delay)
     http_server.close()
 
+@attr('basic_test')
+def test_ps_s3_persistent_topic_configs():
+    """ test persistent topic configurations """
+    conn = connection()
+    zonegroup = 'default'
+
+    # create random port for the http server
+    host = get_ip()
+    http_port = random.randint(10000, 20000)
+
+    # start an http server in a separate thread
+    http_server = StreamingHTTPServer(host, http_port, num_workers=10, delay=0.5)
+
+    # create bucket
+    bucket_name = gen_bucket_name()
+    bucket = conn.create_bucket(bucket_name)
+    topic_name = bucket_name + TOPIC_SUFFIX
+
+    # create s3 topic
+    endpoint_address = 'http://'+host+':'+str(http_port)
+    persistency_ttl = 30
+    endpoint_args = 'push-endpoint='+endpoint_address+'&persistent=true&time_to_live='+str(persistency_ttl)
+    topic_conf = PSTopicS3(conn, topic_name, zonegroup, endpoint_args=endpoint_args)
+    topic_arn = topic_conf.set_config()
+    # create s3 notification
```

Comment:
can you please also invoke the "topic get" admin command, to verify that the new configuration looks ok there